install/docker/centos-docker-install.md

```diff
@@ -37,7 +37,13 @@
 
    2. Docker `daemon.json`
 
-       Add the following definitions to `/etc/docker/daemon.json`:
+      Create docker configuration folder.
+
+      ```bash
```

Comment:
Please can you remove the `bash` so that it is a default code block (` ``` `).

# Explanation

The reason for this being that we actually execute the *bash* blocks when we test changes to this document in our CI. But "Option 2" is not tested as it conflicts with "Option 1". We could just create the directory but I think it's clearer (for the team) if "Option 2" is simply not executed at all.

Reply:
I see, I've removed bash indicator from block.